src/extensions/codemirror/CodeMirror.tsx

```diff
@@ -280,6 +354,7 @@ export const CodeEditor = ({
             id={id ? id : name ? `codemirror-${name}` : undefined}
             ref={parent}
             data-test-id="codemirror-wrapper"
+            data-testid={dataTestId ? `${dataTestId}_codemirror}` : undefined}
```

Comment:
We probably don't need to alter the test id by a suffix, the developer could use `xyz_codemirror` directly.